server/tests/unit/common/fbs/test_matrix.py

```diff
@@ -50,7 +50,7 @@ def test_encode_DataFrame(self):
                 "d": pd.Series(["x", "y", "z", "x", "y", "z", "a", "x", "y", "z"], dtype="category"),
```

Comment:
This file needs to be expanded to cover all of the new FBS encoding types.  I see a bunch of missing cases (suggest looking at code coverage and ensuring you hit 100% on the matrix.py and fbscoders.py files):
* sparse float64
* sparse:  CSR & CSC
* Categoricals with code width 8, 16 and 32
etc.